scripts/common.py

```diff
@@ -132,8 +132,7 @@ def setup():
     if registration_info.status_code != 200:
         fail("This account is not registered with FireCloud.")
 
-    print "Using credentials for firecloud account:", registration_info.json()["userInfo"]["userEmail"]
-
+    sys.stderr.write("Using credentials for firecloud account: %s\n" % registration_info.json()["userInfo"]["userEmail"])
```

Comment:
why this change?

Reply:
yes forgot to document this - i moved some stuff to stderr so that you could pipe some script outputs without getting this info